utils/run-test

```diff
@@ -171,6 +171,7 @@ def main():
         # build_dir.
         paths = args.paths
         if not paths:
+            parser.print_usage()
             error_exit("error: too few arguments")
```

Comment:
well-behaved commands show usage when invalid arguments or `--help` is passed, and exit with nonzero result when the arguments are invalid.